grails-app/domain/com/redhat/theses/Link.groovy

```diff
@@ -0,0 +1,15 @@
+package com.redhat.theses
+
+class Link implements Serializable{
+    String url
+    
+    static mapping = {
+        id composite: ['url']
```

Comment:
Using the same link (say "www.google.com") twice will result in Primary Key Violation.  You should  create a synthetic id field. It may look strange at first but it has other benefits -- for example you may want to ad other attributes (such as description) to the Link entity.